### PR TITLE
added the ability to specify starting address in command line

### PR DESCRIPTION
**Pull request by brendabrandy**

### CODE REVIEW

File: kstool/kstool.cpp [hunk 1]
```diff
@@ -77,13 +77,14 @@ int main(int argc, char **argv)
     ks_engine *ks;
     ks_err err = KS_ERR_ARCH;
     char *mode, *assembly = NULL;
+    uint64_t start_addr;
```
Comment (aquynh) [inline kstool/kstool.cpp]: it is better to initialize `start_address = 0` here, so it becomes default value, even when `argc == 2` (which you are missing now). then `start_address = 0` can be removed.

File: kstool/kstool.cpp [hunk 2]
```diff
@@ -113,13 +114,18 @@ int main(int argc, char **argv)
...
-        mode = argv[1];
-        assembly = argv[2];
-    } else {
+        mode = argv[1];		//the architecture would be argument 1
+        assembly = argv[2];	//the assembly string would be argument 2
+	start_addr = 0;
+    } else if (argc == 4) {
+	mode = argv[1];
+	assembly = argv[2];
+	start_addr = strtoull(argv[3], NULL,16);
```
Comment (aquynh) [inline kstool/kstool.cpp]: do you want to handle error when strtoull() fails here?

File: kstool/kstool.cpp [hunk 3]
```diff
@@ -113,13 +114,18 @@ int main(int argc, char **argv)
...
+    }else {
```
Comment (aquynh) [inline kstool/kstool.cpp]: please put a space after `{`

File: kstool/kstool.cpp [hunk 4]
```diff
@@ -113,13 +114,18 @@ int main(int argc, char **argv)
...
+    } else if (argc == 3 || argc == 4) {//assembly + line to be loaded
 #else
     if (argc == 3) {
 #endif
-        mode = argv[1];
-        assembly = argv[2];
-    } else {
+        mode = argv[1];		//the architecture would be argument 1
+        assembly = argv[2];	//the assembly string would be argument 2
+	start_addr = 0;
```
Comment (aquynh) [inline kstool/kstool.cpp]: wrong indentation here?

**Comment by aquynh** [comment]
will you fix these issues

**Comment by brendabrandy** [comment]
Yes give me a week or so... I am kinda swamped this week.

**Comment by aquynh** [comment]
merged, and fixed those issues i mentioned. thanks!

**Pull request merged**